docs/snippets/ie_common.cpp

```diff
@@ -50,10 +50,34 @@ int main() {
     }
     //! [ie:get_input_tensor]
 
+    //! [ie:set_callback]
+    infer_request.SetCompletionCallback<std::function<void(InferenceEngine::InferRequest, InferenceEngine::StatusCode)>>(
+        [&](InferenceEngine::InferRequest request, InferenceEngine::StatusCode status) {
+            if (status != InferenceEngine::OK) {
+                // Process error code
+            } else {
+                // Extract inference result
+                InferenceEngine::Blob::Ptr output_blob = request.GetBlob(outputs.begin()->first);
+                // Restart inference if needed
+                // request.StartAsync();
```

Comment:
I think we can run a new inference in the snippet
```suggestion
                 request.StartAsync();
```

Reply:
added